_topic_map.yml

```diff
@@ -115,13 +115,19 @@ Topics:
     File: installing-gcp-customizations
   - Name: Uninstalling a cluster on GCP
     File: uninstalling-cluster-gcp
-#- Name: Installing in a disconnected environment
-#  Dir: installing_disconnected
-#  Topics:
+- Name: Installing in restricted networks
+  Dir: installing_restricted_networks
+  Topics:
 #  - Name: Preparing for a disconnected installation
-#    File: installing-disconnected-preparations
-#  - Name: Installing in a disconnected environment
-#    File: installing-disconnected
+#    File: installing-restricted-networks-preparations
+  - Name: Restricted network AWS installation
+    File: installing-restricted-networks-aws
+  - Name: Restricted network bare metal installation
```

Comment:
@morenod Could you help review this part - upi install on baremetal in a restricted network (originally we call it 'disconnected')?

Reply:
On "Sample install-config.yaml file for bare metal". Point 14: imageContentSources is a requirement?

What happens if quay.io connection is also not allowed from the host where installer is being executed and image have been already uploaded to the internal registry?

Reply:
You still need to provide imageContentSources with the mirror mapping of quay.io and the internal registry to the install-config.yaml. This way you don't have to go in and manually change the names of the images to be `internal-registry/internal-repo`, it can still be `quay.io/myrepo` and with the mirror mapping, the installer will know to try the mirrored location (i.e the internal registry) first.